sdk/storage/storage-blob/review/storage-blob.api.ts

```diff
@@ -0,0 +1,1424 @@
+import * as Models from "./generated/src/models";
+
+// @public
+class AccountSASPermissions {
+  add: boolean;
+  create: boolean;
+  delete: boolean;
+  list: boolean;
+  static parse(permissions: string): AccountSASPermissions;
+  process: boolean;
+  read: boolean;
+  toString(): string;
```

Comment:
I wonder about using `toJSON()` instead of `toString()` for these kinds of things. Maybe @bterlson has an opinion?

Reply:
This actually returns a permission string like 'rwdc'

Reply:
toString() seems appropriate here.